test/integration/scheduler/queue_test.go

```diff
@@ -507,6 +516,61 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod2"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
+			name:         "Pod rejected with node by the VolumeZone plugin is requeued when the label of PV binding to the pod is updated",
```

Comment:
I think we need more test cases for this plugin, especially on events: PVAdd,  PVCAdd/Update, StorageClassAdd. 

Please also add a second unscheduled pod that still won't be requeued after the event happens.